.github/scripts/release.py

```diff
@@ -185,6 +193,16 @@ def main():
     release = JulesDocsRelease(vnumber)
 
     try:
+        if args.gh_pages:
+            # Create a new gh-pages branch and replace the contents
+            # with the current trunk before checking it out
+            cmd = ["git", "branch", "-C", "gh-pages"]
```

Comment:
curious about creating a new gh-pages branch each time?  I'd have assumed this would be a preexisiting thing we'd want to checkout and update rather than create?

Reply:
Good point.  The more I think about my approach, the more convinced I am it's not quite right.  Instead of resetting and copying master to gh-pages every time, it would be better to merge from master to gh-pages otherwise we will lose previous copies of the docs because they won't have been added to the master.

I'll modify the script and do some testing with a fork of the jules docs repo.